apps/laboratory/tests/canary.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { DEFAULT_SESSION_PARAMS } from './shared/constants'
+import { testMW } from './shared/fixtures/w3m-wallet-fixture'
+
+testMW.beforeEach(
+  async ({ modalPage, walletPage, modalValidator, walletValidator, browserName }) => {
+    // Canary doesn't need all platforms
+    if (browserName !== 'chromium' || modalPage.library !== 'ethers') {
+      return
+    }
+    await modalPage.copyConnectUriToClipboard()
+    await walletPage.connect()
+    await walletPage.handleSessionProposal(DEFAULT_SESSION_PARAMS)
+    await modalValidator.expectConnected()
+    await walletValidator.expectConnected()
+  }
+)
+
+testMW.afterEach(async ({ modalPage, modalValidator, walletValidator, browserName }) => {
+  // Canary doesn't need all platforms
+  if (browserName !== 'chromium' || modalPage.library !== 'ethers') {
+    return
+  }
+  await modalPage.disconnect()
+  await modalValidator.expectDisconnected()
+  await walletValidator.expectDisconnected()
+})
+
+testMW(
+  'it should sign',
+  async ({ modalPage, walletPage, modalValidator, walletValidator, browserName }) => {
+    // Canary doesn't need all platforms
+    if (browserName !== 'chromium' || modalPage.library !== 'ethers') {
+      testMW.skip()
```

Comment:
Why skip here and empty return on the others?

Reply:
It's an intricacy of Playwright